org.freedesktop.fldigi.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>org.freedesktop.fldigi</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>Fldigi</name>
+ <summary>Amateur Radio Sound Card Communications</summary>
+ <developer_name>dave-w1hkj, mdblack98, sb42</developer_name>
+ <url type="homepage">https://sourceforge.net/projects/fldigi/
```

Comment:
this tag isn't closed so the xml is invalid and it fails to compile properly.

Reply:
ok